simulator/simple_planning_simulator/src/simple_planning_simulator/vehicle_model/sim_model_delay_steer_acc.cpp

```diff
@@ -127,7 +128,7 @@ Eigen::VectorXd SimModelDelaySteerAcc::calcModel(
   Eigen::VectorXd d_state = Eigen::VectorXd::Zero(dim_x_);
   d_state(IDX::X) = vel * cos(yaw);
   d_state(IDX::Y) = vel * sin(yaw);
-  d_state(IDX::YAW) = vel * std::tan(steer) / wheelbase_;
+  d_state(IDX::YAW) = vel * std::tan(steer + steer_bias_) / wheelbase_;
```

Comment:
Only yaw among the states has a biased value. each of the states should be unified as to whether it is an actual value or a sensor value.